nfs/parse_nfs_test.go

```diff
@@ -32,6 +32,131 @@ func TestNewNFSdClientRPCStats(t *testing.T) {
 			name:    "invalid file",
 			content: "invalid",
 			invalid: true,
+		}, {
+			name: "good old kernel version file",
+			content: `net 70 70 69 45
+rpc 1218785755 374636 1218815394
+proc2 18 16 57 74 52 71 73 45 86 0 52 83 61 17 53 50 23 70 82
+proc3 22 0 1061909262 48906 4077635 117661341 5 29391916 2570425 2993289 590 0 0 7815 15 1130 0 3983 92385 13332 2 1 23729
+proc4 48 98 51 54 83 85 23 24 1 28 73 68 83 12 84 39 68 59 58 88 29 74 69 96 21 84 15 53 86 54 66 56 97 36 49 32 85 81 11 58 32 67 13 28 35 90 1 26 1337
+`,
+			stats: &nfs.ClientRPCStats{
+				Network: nfs.Network{
+					NetCount:   70,
+					UDPCount:   70,
+					TCPCount:   69,
+					TCPConnect: 45,
+				},
+				ClientRPC: nfs.ClientRPC{
+					RPCCount:        1218785755,
+					Retransmissions: 374636,
+					AuthRefreshes:   1218815394,
+				},
+				V2Stats: nfs.V2Stats{
+					Null:     16,
+					GetAttr:  57,
+					SetAttr:  74,
+					Root:     52,
+					Lookup:   71,
+					ReadLink: 73,
+					Read:     45,
+					WrCache:  86,
+					Write:    0,
```

Comment:
Nit: you can omit all the zeros from all of these test structures if you want.

Reply:
I'd rather keep them in as explicit.